packages/nextjs/app/builders/0x8eE31084d2914fA84Baae3460093564934837898/page.tsx

```diff
@@ -0,0 +1,94 @@
+import Image from "next/image";
+import { NextPage } from "next";
+import type { Metadata } from "next";
```

Comment:
You can remove a bunch of imports that are not being used!

![image](https://github.com/user-attachments/assets/f6d671c1-9dbc-4b55-bf6e-30cdb8786ae4)